lib/helper/sanitizeNumber.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * parses a non number value and returns the default
+ * in case NaN.
+ * @param {*} num 
+ * @param {number} defaultVal 
+ */
+module.exports = function sanitizeNumber(num, defaultVal) {
+  const parsed = parseInt(num, 10);
+  return Number.isNaN(parsed) ? defaultVal : parsed;
```

Comment:
Should be greater than 0 as well. You handle 0 here but not negative numbers.

Reply:
Ya, I thought about it. When testing negative number, I just want the server to send an error.
What are your thoughts on sending an error or correcting it for the user. ?

Reply:
yea, that makes sense. I'm guessing that a 500 error which we'll want to write alarms against so anyplace where we can stop them from happening is best. Here we should return a 400 error as its bad use input.